includes/plugins/woocommerce/my-account/class-my-account-ui-v1.php

```diff
@@ -19,10 +19,52 @@ class My_Account_UI_V1 {
 	 * @codeCoverageIgnore
 	 */
 	public static function init() {
+		\add_filter( 'page_template', [ __CLASS__, 'page_template' ] );
+		\add_filter( 'newspack_ads_should_show_ads', [ __CLASS__, 'suppress_ads' ] ); // Suppress ads on My Account pages.
+		\add_filter( 'newspack_popups_assess_has_disabled_popups', [ __CLASS__, 'suppress_popups' ] ); // Suppress popups on My Account pages.
 		\add_filter( 'body_class', [ __CLASS__, 'add_body_class' ] );
-		\add_filter( 'do_shortcode_tag', [ __CLASS__, 'add_newspack_ui_wrapper' ], 10, 2 );
 		\add_action( 'wp_enqueue_scripts', [ __CLASS__, 'enqueue_assets' ], 11 );
 		\add_filter( 'wc_get_template', [ __CLASS__, 'wc_get_template' ], 10, 5 );
+		\add_filter( 'woocommerce_account_menu_items', [ __CLASS__, 'my_account_menu_items' ], 1001 );
+	}
+
+	/**
+	 * Render My Account pages with a no-header/no-footer page template.
+	 *
+	 * @param string $template The template.
+	 * @return string The template file path.
+	 */
+	public static function page_template( $template ) {
+		if ( function_exists( 'is_account_page' ) && \is_account_page() && \is_user_logged_in() ) {
+			return __DIR__ . '/templates/v1/myaccount.php';
```

Comment:
```suggestion
			return __DIR__ . '/templates/v1/my-account.php';
```

I think we should follow Woo's naming here.

Reply:
They actually use both `myaccount` and `my-account` in various places, but notably `myaccount` for [template parts](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce/templates/myaccount) vs `my-account` for the shortcode and permalinks. Not sure if there's any logic behind the difference, though. I'm okay with picking one and sticking to it in our code, though.

Reply:
I was thinking of the [`my-account.php` file](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/templates/myaccount/my-account.php), but you're right, they also use `myaccount` for the directory 🤔 

I'll let you decide 😄

Reply:
efc976c and 9c9c2d9 remove the `myaccount-` prefixes from the custom template parts—the prefix is redundant IMO since they're inside a `my-account` directory now. The `myaccount.php` custom page template is now `my-account.php` to match our own naming convention.